.github/draft_release_notes.sh

```diff
@@ -0,0 +1,141 @@
+#! /bin/bash
+# Copyright The Shipwright Contributors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# this script assumes the GITHUB_TOKEN and PREVIOUS_TAG environment variables have been set;
+# it produces a 'Changes.md' file as its final output;
+# the file 'last-300-prs-with-release-note.txt' that is produces is intermediate data; it is not
+# pruned for now to assist development of the release notes process (we are still curating all this)
+
+if [ -z ${GITHUB_TOKEN+x} ]; then
+  echo "Error: GITHUB_TOKEN is not set"
+fi
+if [ -z ${PREVIOUS_TAG+x} ]; then
+  echo "Error: PREVIOUS_TAG is not set"
+fi
```

Comment:
Should it `exit 1` when the expected variables are not set?

Reply:
Not needed, the inputs are mandatory
```
inputs:
      release:
        description: 'Desired tag'
        required: true 
      tags:
        description: 'Previous tag'
        required: true
```